.gitignore

```diff
@@ -4,5 +4,6 @@
 /src/*/*.tar.gz
 /src/*/*.tgz
 /src/*/*.zip
+/publish_dist_sftp_batch
```

Comment:
ca we have this file in build-pythia/publish_dist_sftp_batch and then we will no longer need to add extra ignore?

Reply:
Yes, seems like a good idea, thanks!